cmd/state/main.go

```diff
@@ -146,11 +146,13 @@ func run(args []string, out output.Outputer) (int, error) {
 	}
 	// Set up conditional, which accesses a lot of primer data
 	sshell := subshell.New()
-	conditional := constraints.NewPrimeConditional(pjOwner, pjName, pjNamespace, sshell.Shell())
+	auth := authentication.Get()
+	conditional := constraints.NewPrimeConditional(auth, pjOwner, pjName, pjNamespace, sshell.Shell())
 	project.RegisterConditional(conditional)
+	project.RegisterExpander("mixin", project.NewMixin(auth).Expander)
```

Comment:
Since this is all contained within the project package there's no reason to register this externally. Just hook it up via http://github.com/ActiveState/cli/blob/8182de4063d5524abb67c1fefb10da02178d8050/pkg/project/registry.go#L20-L20

Reply:
In that case I should probably not initialize with `auth`, as this would be done in the package's `init()` function.  That is the reason I put it here - to avoid using the `authentication.Get()` call...

Reply:
Ok, let's leave it here for now. We can revisit once we refactor the project package.